conformance/tests/basic/inferencepool_accepted.yaml

```diff
@@ -0,0 +1,27 @@
+# Basic InferencePool for acceptance testing.
+# This manifest defines the minimal required fields to create a valid
+# InferencePool resource, which the InferencePoolAccepted test will use
+# to verify that the controller recognizes and accepts the resource.
+
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferencePool
+metadata:
+  # This name must match the 'poolNN' variable defined in the
+  # conformance/tests/basic/inferencepool_accepted.go test file.
+  name: inferencepool-basic-accepted
+  # This namespace should be one created by the base manifests.
+  namespace: default
+spec:
+  # --- Selector (Required) ---
+  # Selects the Pods belonging to this pool.
+  selector:
+    app: "vllm-llama3-8b-instruct"
```

Comment:
Why is `app: "vllm-llama3-8b-instruct"` being used when the base manifest uses `app: infra-backend-v1`?

Reply:
Done. This is a place holder tests I had the values set based on local env. Updated to match the manifest.